walkthroughs/howto-ingress-enhancements/infrastructure/ecs-service-with-alb.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Should this file name be `ecs-service-with-nlb`?

Reply:
Both of these were for when we were experimenting with using an ALB instead. They can be deleted.